app/main.py

```diff
@@ -1,2 +1,53 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length: int = 0
+        self.capacity: int = 8
+        self.load_factor: float = 0.66
+        self.hash_table: list = [None] * self.capacity
+
+    def __add_to_hash_table(self, key_hash: int, key: Any, value: Any) -> None:
+        _index = key_hash % self.capacity
+        while True:
+            if not self.hash_table[_index]:
+                self.hash_table[_index] = (key_hash, key, value)
+                self.length += 1
+                break
+            elif self.hash_table[_index][1] == key:
+                self.hash_table[_index] = (key_hash, key, value)
```

Comment:
I know it doesn't make a difference in this case, but let's imagine that some other person will use this dict. ((key, hash, value)